drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/plugins/ActionSetFactPluginTest.java

```diff
@@ -225,7 +208,7 @@ public void testIsValidWhenItIsValid() {
         when(presenter.getModel()).thenReturn(model);
         when(actionCol52.getHeader()).thenReturn("header");
         when(actionWrapper.getFactField()).thenReturn("factField");
-        when(actionWrapper.getFactType()).thenReturn("factType");
+        when(actionWrapper.getFactType()).thenReturn("");
```

Comment:
Hi @karreiro I would like to ask about this change. It the insert fact action really valid if the inserted fact type equals to empty string? I would expect that the insert fact action must handle with some specific fact type like `Person`. Or am I missing something?